about/pricing/strategy.md

```diff
@@ -15,13 +26,14 @@ Our prices should:
 - Be competitive with other "Data Science environment as a service" offerings (see below for how we consider ourselves relative to similar offerings).
 - Be sustainable for the communities we serve, with mechanisms to accommodate institutions with fewer resources.
 
+
 ## Base fees for three service types
 
 These are based on major use-cases in the communities we have served.
 
 - **Educational communities** need basic hubs that are reliable, secure, and that serve a community of users. They tend to charge course fees and can recoup some costs this way. Assume 22 educational hubs per engineer.
```

Comment:
```suggestion
## Pricing structure

Currently, we base our pricing on two major items:

### Flat monthly fees

We charge a flat monthly fee to cover [our personnel costs](costs/people.md).
We estimate the number of hubs an engineer can run, and use this to estimate our costs per hub after adding in project management and administration costs.

Most hubs take extra effort during the _set up_ phase, and relatively less effort to maintain over time (depending on how many change requests a community makes).
As such we suspect that this pricing structure does not cover our costs in the first month or two, but regains those costs in subsequent months.

In the future we may try to perform a more nuanced mapping of costs onto effort from our team, but for now we wish to keep things simple and predictable while we learn more.

### Pass-through cloud costs

In addition to our monthly hub fees, we pass cloud costs [directly to the communities we serve](costs/cloud.md), without taking any percentage markup.
We do this for two reasons:

1. In our eyes, we are running infrastructure _on behalf of each community_, and wish to act as if a member of that community were running the infrastructure themselves. We are simply being compensated for our time and expertise.
2. Adding a percentage markup on cloud costs may create perverse incentives for us to avoid optimizing down a community's cloud costs.

For these reasons, we are currently [passing through cloud costs directly to communities](costs/cloud.md).

## Base fees for three service types

These are based on major use-cases in the communities we have served.

- **Educational communities** need basic hubs that are reliable, secure, and that serve a community of users. They tend to charge course fees and can recoup some costs this way. Assume 22 educational hubs per engineer.
```